ethereum/sign.go

```diff
@@ -0,0 +1,38 @@
+package ethereum
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+// ECSignature contains the parameters of an elliptic curve signature
+type ECSignature struct {
+	V byte
+	R []byte
```

Comment:
If `V` and `R` are always a fixed length, we should use an array (e.g. `[32]byte`) instead of a slice here.

Reply:
Agreed!